### PR TITLE
Todo request api

### PR DESCRIPTION
**Pull request by NataliaHuberna**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,32 +2,97 @@ import React, { Component } from 'react';
 import './App.css';
 import TaskInput from './components/TaskInput/TaskInput';
 import List from './components/List/List';
+import { API_URL } from './constants/constantsApi';
```
Comment (nataliazhuravlova) [inline src/App.js]: constantsApi -> api

File: src/App.js [hunk 2]
```diff
@@ -2,32 +2,97 @@ import React, { Component } from 'react';
...
-  render() {
+    hideLoader = () => this.setState({ isLoading: false });
+
+    getDataTasks = async () => {
+        try {
+            this.showLoader();
+            const { data } = await axios.get(`${API_URL}?_limit=7`);
+            this.setState({ todos: data });
+        } catch (error) {
+            console.log(error);
```
Comment (nataliazhuravlova) [inline src/App.js]: unnessasary catch

File: src/App.js [hunk 3]
```diff
@@ -2,32 +2,97 @@ import React, { Component } from 'react';
...
+        }
+    };
+
+    addTask = async (title) => {
+        try {
+            this.showLoader();
+            const body = { title, id: Date.now(), completed: false };
+            const { data } = await axios.post(API_URL, body);
+            this.setState({ todos: [...this.state.todos, body] });
+        } catch (error) {
```
Comment (nataliazhuravlova) [inline src/App.js:44]: the same

File: src/App.js [hunk 4]
```diff
@@ -2,32 +2,97 @@ import React, { Component } from 'react';
...
+            console.log(error);
+        } finally {
+            this.hideLoader();
+        }
+    };
+
+    checkChange = (id) => {
+        this.setState({
+            todos: this.state.todos.map((element) =>
```
Comment (nataliazhuravlova) [inline src/App.js]: one row

File: src/App.js [hunk 5]
```diff
@@ -2,32 +2,97 @@ import React, { Component } from 'react';
...
+                    : element
+            ),
+        });
+    };
+
+    deleteTask = async (id) => {
+        try {
+            this.showLoader();
+            const response = await axios.delete(`${API_URL}/${id}`);
+            if (response.status === 200) {
```
Comment (nataliazhuravlova) [inline src/App.js]: do not check status

File: src/components/Loader/Loader.js [hunk 6]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import './Loader.css';
+
+const Loader = () => {
+    return (
```
Comment (nataliazhuravlova) [inline src/components/Loader/Loader.js]: unnecessary return

Comment (NataliaHuberna) [reply src/components/Loader/Loader.js]: if I don't do "retern", then todo doesn't work

**Pull request closed without merging**